classes/ImageManager.php

```diff
@@ -475,7 +475,12 @@ public static function validateUpload($file, $maxFileSize = 0, $types = null, $m
             return Context::getContext()->getTranslator()->trans('Image is too large (%1$d kB). Maximum allowed: %2$d kB', [$file['size'] / 1024, $maxFileSize / 1024], 'Admin.Notifications.Error');
         }
         if (!ImageManager::isRealImage($file['tmp_name'], $file['type'], $mimeTypeList) || !ImageManager::isCorrectImageFileExt($file['name'], $types) || preg_match('/\%00/', $file['name'])) {
-            return Context::getContext()->getTranslator()->trans('Image format not recognized, allowed formats are: .gif, .jpg, .png', [], 'Admin.Notifications.Error');
+            $typesTxt = $types ? '.' . implode(', .', $types) : '.gif, .jpg, .png';
+            return Context::getContext()->getTranslator()->trans(
```

Comment:
```suggestion

            return Context::getContext()->getTranslator()->trans(
```

Add a break line before the return